pkg/appvalidation/business_case.go

```diff
@@ -27,7 +27,7 @@ func checkUniqLifecycleCosts(costs models.EstimatedLifecycleCosts) (string, stri
 
 // check the system intake status is submitted
 func checkSystemIntakeSubmitted(intake *models.SystemIntake) (string, string) {
-	if intake.RequestFormState == models.SIRFSSubmitted || intake.Status != models.SystemIntakeStatusINTAKEDRAFT {
+	if intake.RequestFormState == models.SIRFSSubmitted {
```

Comment:
this needs to be looked at as it modifies a conditional

Reply:
I think this is right. The v1 status here was looking to see if the intake was progressed out of the initial request step. Now we're just making sure that the request form was submitted.

However, since [the usage](https://github.com/CMSgov/easi-app/blob/7c7a7cca711dab2d46d58f5068e19c750c9dbfaf/pkg/appvalidation/business_case.go#L52) suggests that perhaps we should be making sure the request was submitted and the step is draft biz case? Thoughts @ClayBenson94?

Reply:
Yeah, I took a peek at how this function was used and came to a similar conclusion.

I don't believe that, if an admin progresses to a step _before_ a user re-submits an intake form that the intake form will be marked as submitted.

For that reason, I almost wonder if we should just replace this check entirely with `if intake.Step != SystemIntakeStepINITIALFORM`? (That way, if they progress to either `draft` or `final`, it's a valid operation to create the biz case)

Though, taking a step back even further, it makes me wonder what the real harm is in preventing a business case from being created at _any_ point, but for the sake of keeping things from changing too much, perhaps my suggestion above is a good middle ground?

@mynar7 @samoddball

Reply:
I think you're right, we should just make sure the initial form is completed. We could also check for `intake.SubmittedAt` to make sure it has been submitted at least once.

Side note: As I understand it we can only have 1 business case per intake, correct? Draft/Final reference the same data in the backend, but we seem to have no guards against creating multiple business cases.